webapp/src/components/Vault/VaultActionsForm/EarnModal/PreviewStep.tsx

```diff
@@ -22,17 +22,105 @@ import TooltipExplanation from "shared/lib/components/Common/TooltipExplanation"
 import useLoadingText from "shared/lib/hooks/useLoadingText";
 import useUSDC, { DepositSignature } from "../../../../hooks/useUSDC";
 import useEarnStrategyTime from "../../../../hooks/useEarnStrategyTime";
+import { fadeIn } from "shared/lib/designSystem/keyframes";
 
-const FormTitle = styled(Title)`
+const Logo = styled.div<{ delay?: number; show?: boolean }>`
+  margin-top: -40px;
+  margin-bottom: -40px;
+
+  ${({ show, delay }) => {
+    return (
+      show &&
+      css`
+        opacity: 0;
+        animation: ${fadeIn} 1s ease-in-out forwards;
+        animation-delay: ${delay || 0}s;
+      `
+    );
+  }}
+`;
+
+const InfoPreview = styled.div<{ delay?: number; show?: boolean }>`
+  ${({ show, delay }) => {
+    return (
+      show &&
+      css`
+        opacity: 0;
+        animation: ${fadeIn} 1s ease-in-out forwards;
+        animation-delay: ${delay || 0}s;
+      `
+    );
+  }}
+`;
+
+const FormTitle = styled(Title)<{ delay?: number; show?: boolean }>`
   font-size: 22px;
   line-height: 28px;
   letter-spacing: 1px;
+  text-align: center;
+
+  ${({ show, delay }) => {
+    return (
+      show &&
+      css`
+        opacity: 0;
+        animation: ${fadeIn} 1s ease-in-out forwards;
+        animation-delay: ${delay || 0}s;
+      `
+    );
+  }}
 `;
 
-const WarningContainer = styled.div<{ color: string }>`
+const WarningContainer = styled.div<{
+  color: string;
+  delay?: number;
+  show?: boolean;
+}>`
   padding: 8px;
   background: ${(props) => props.color}14;
   border-radius: ${theme.border.radiusSmall};
+
+  ${({ show, delay }) => {
+    return (
+      show &&
+      css`
+        opacity: 0;
+        animation: ${fadeIn} 1s ease-in-out forwards;
+        animation-delay: ${delay || 0}s;
+      `
+    );
+  }}
+`;
+
+const DetailRow = styled.div<{ delay?: number; show?: boolean }>`
+  ${({ show, delay }) => {
+    return (
+      show &&
+      css`
+        opacity: 0;
+        animation: ${fadeIn} 1s ease-in-out forwards;
+        animation-delay: ${delay || 0}s;
+      `
+    );
+  }}
+`;
+
+const FormButton = styled(ActionButton)<{ delay?: number; show?: boolean }>`
+  ${({ show, delay }) => {
+    return (
+      show &&
```

Comment:
i think we should do `show ? ... : ""` instead

this will return `false` if show is not `true`, don't think it causes any problems visually rn, but better to do the alternative